feature-sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Sessions.kt

```diff
@@ -458,3 +498,48 @@ fun SessionsLoadingPreview() {
         )
     }
 }
+
+@OptIn(ExperimentalPagerApi::class)
+@Composable
+fun rememberPagerContentsScrollState(
+    pagerState: PagerState,
+    timetableStates: List<TimetableState>,
+    sessionsListListStates: List<LazyListState>
+): PagerContentsScrollState = remember {
+    PagerContentsScrollState(pagerState, timetableStates, sessionsListListStates)
+}
+
+@OptIn(ExperimentalPagerApi::class)
+@Stable
+class PagerContentsScrollState(
+    val pagerState: PagerState,
+    val timetableStates: List<TimetableState>,
+    val sessionsListStates: List<LazyListState>
+) {
+    private var _scrollingPage = pagerState.currentPage
+
+    private val _isScrollTop = mutableStateOf(true)
+    val isScrollTop get() = _isScrollTop.value
+
+    suspend fun scrollTimetable() {
```

Comment:
I don't think we need suspend modifier here 👀

Reply:
Oh I see... That's right！